octal/cavium_env/rf_card_cal_tip/test_report/test_20181219_153212.txt

```diff
@@ -0,0 +1,2 @@
+./test_report/test_20181219_153212.txt
```

Comment:
Unless you want to include test results as an example, you should probably add these to .gitignore, or move them before commit

Reply:
yes these were stray adds, deleted in later commit